.github/workflows/build.yml

```diff
@@ -375,6 +375,8 @@ jobs:
         shell: msys2 {0}
         run: |
           cmake --install ${{ env.BUILD_DIR }}
+          touch ${{ env.INSTALL_DIR }}/manifest.txt
```

Comment:
Isn't this supposed to only be for portable builds?

Reply:
the products of this step are used in portable builds. the manifest will just get left out of the installer or serve as intended in non installer packages.

Reply:
yep, that's perfectly ok and frankly intended imo. a manifest is a manifest. it may not have an explicit use but it may be helpful to someone.